05/rodne_cislo_finish.py

```diff
@@ -0,0 +1,50 @@
+# Input ID number in Czech Republic
+# write ouf corect/incorect ID
+
+
+# MONTHS 30 DAYS
+months_30 = [4, 6, 9, 11]
+
+# MONTHS 31 DAYS
+months_31 = [1, 3, 5, 7, 8, 10, 12]
+
+# YEARS
+years = list(range(86, 100)) + list(range(00, 23))
+
+while True:
+
+    ID = input('Napiš rodné číslo: ')
+    if len(ID) != 11:
+        print("Nesprávná délka rodného čísla.")
+        continue
+
+    only_numbers = (ID.replace('/', ''))
+    if not only_numbers.isdigit() and ID[6] != "/":
+        print("Nesprávný formát.")
+        continue
+
```

Comment:
Zkoušela jsem převést only_numbers jednorázově na int, ale to pak nejdou vybírat jednotlivé části, protože to není pro int podporované.  A nebo nechápu jak jsi to myslela :) proto jsem řádky 26 - 30 je upravila . Šlo by nevytvářet proměnné a dát to tam rovnou např. int(only_numbers[:2]), jenže to mi přišlo míň přehledné než vytvořit proměnnou

Reply:
Luci, tu som teraz trochu stratená. Nevidím zmeny, ani moje pôvodné komentáre a pri niekoľkých verziách ktoré som videla sa v tom už neorientujem :sweat_smile:

Reply:
zkus mrknout sem:) https://github.com/LuckaSokolka/pyladies/pull/6#discussion_r1014726236

Reply:
a tady jde vidět ten upravený kód 05/rodne_cislo_finish.py

Reply:
https://github.com/LuckaSokolka/pyladies/commit/92498d3b7f2f11f0cbfd4439d2937c5f74d35a07

Reply:
Jasné, už to vidím, pravdu máš, z integeru slicovať nejde :+1: